src/arqa.py

```diff
@@ -117,6 +118,112 @@ def arc_fit_qa(slf, fit, outfil=None, ids_only=False, title=None):
     return
 
 
+def flexure(slf, det, flex_dict):
+    """ QA on flexure measurement
+
+    Parameters
+    ----------
+    slf
+    det
+    flex_dict
+
+    Returns
+    -------
+
+    """
+    # Setup
+    nobj = len(slf._specobjs[det-1])
+    if nobj == 0:
+        return
+    ncol = min(3,nobj)
+    nrow = nobj // ncol + ((nobj%ncol) > 0)
+
+
+    plt.figure(figsize=(8, 5.0))
+    plt.clf()
+    gs = gridspec.GridSpec(nrow, ncol)
+
+    # Correlation QA
+    for o,specobj in enumerate(slf._specobjs[det-1]):
+        ax = plt.subplot(gs[o//ncol, o%ncol])
+        # Fit
+        fit = flex_dict['polyfit'][o]
+        xval = np.linspace(-10., 10, 100) + flex_dict['corr_cen'][o] + flex_dict['shift'][o]
+        model = (fit[2]*(xval**2.))+(fit[1]*xval)+fit[0]
+        mxmod = np.max(model)
+        ylim = [np.min(model/mxmod), 1.5]
+        ax.plot(xval-flex_dict['corr_cen'][o], model/mxmod, 'k-')
+        # Measurements
+        ax.scatter(flex_dict['subpix'][o]-flex_dict['corr_cen'][o],
+                   flex_dict['corr'][o]/mxmod, marker='o')
+        # Final shift
+        ax.plot([flex_dict['shift'][o]]*2, ylim, 'g:')
+        # Label
+        ax.text(0.04, 0.25, '{:s}'.format(specobj.idx), transform=ax.transAxes, size='large', ha='left')
```

Comment:
0.25 and 0.15 will almost always overlap the model curve. Perhaps values closer to 0.85 and 0.75 would be better?